packages/fluentui/docs/src/views/Accessibility.tsx

```diff
@@ -48,7 +48,6 @@ export default () => (
       <li>{link('High Contrast', '#high-contrast')}</li>
       <li>{link('Zoom', '#zoom')}</li>
     </ul>
-
```

Comment:
Can we avoid all these line removals? Is this some formatting issue

Reply:
I would like to avoid but not sure how? :)
when I switch off formatting these lines were removed when I did commit.

Reply:
issue was that I didn't wrap text in <p> tag